e2e/src/app.e2e-dealList.ts

```diff
@@ -0,0 +1,84 @@
+import { element, browser, by } from 'protractor';
+import { AppPage } from './app.po';
+
+describe('workspace-project App', () => {
+  let page: AppPage;
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+  describe('Main Page', function() {
+    it('should display table containing list of deals', async function() {
+      // Given
+      await browser.get('http://localhost:4200');
+
+      // When
+      const dealsTable = element(by.css('#dealsTable')).isDisplayed();
+      const newDeal = element(by.css('#newDealB')).isDisplayed();
+
+      // Then
+      expect(await dealsTable).toBeTruthy();
+      expect(await newDeal).toBeTruthy();
+     });
+
+     it('should have 5 rows of deals showing', async function() {
+      // Given
+      await browser.get('http://localhost:4200');
+      await element(by.css('#deleteButton')).isEnabled();
+
+      // When
+      const elements = element.all(by.css('#dealsTable tr'));
+
+      // Then
+      expect(elements.count()).toEqual(5);
+     });
+
+     it('should have 11 columns of deal entities showing', async function() {
+      // Given
+      await browser.get('http://localhost:4200');
+      await element(by.css('#deleteButton')).isEnabled();
+
+      // When
+      const elements = element.all(by.css('#dealsTable th'));
+
+      // Then
+      expect(elements.count()).toEqual(11);
+     });
+
+     it('should go into a particular deals details', async function() {
+      // Given
+      await browser.get('http://localhost:4200');
+
+      // When
+      const elements = element.all(by.css('#link')).get(0);
+
+      // Then
+      elements.click().then(function() {
+      expect(browser.getCurrentUrl()).toEqual('http://localhost:4200/deals/02B4EFAD4432');
+      });
+    });
+
+    it('Should be able to delete a deal', async function() {
+       // Given
```

Comment:
Nice 👍 
You should check if it navigates to the new location (List page before deleting). Also make sure its not available afterwards.